src/screens/Booking/components/BaggageItem/BaggageItem.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {View, Text} from 'react-native';
+import styles from './BaggageItem.styles';
+import {Checkbox} from 'react-native-paper';
+import {MaterialCommunityIcons} from '@expo/vector-icons';
+
+interface BaggageItemProps {
+  value: Boolean;
+  setValue: () => void;
+  icon: any;
```

Comment:
it should be any? it look like a string or undefined for the MaterialCommunityIcons interface in the name prop.